src/components/common/header/PopUp.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export const PopUp =({popupdisplay, popupFn})=>{ 
+    
+    const handleSubmit =(e)=>{
+       popupFn("popupdisplayHIDE");
+    } 
+    const handleClick =(e)=>{
+        e.preventDefault(); 
+         popupFn("popupdisplayHIDE")
+    }
+    return(
+    <div className={popupdisplay} onSubmit={(e) => handleClick(e)}> 
+            
```

Comment:
onSubmit can be removed also as we have no control over the mailerlite form.

I think what can be done instead is add an exit "x" button to close out the popup instead.